qiskit_experiments/randomized_benchmarking/interleaved_rb_analysis.py

```diff
@@ -89,51 +81,98 @@ def fit_fun_interleaved(x, a, _, alpha_int, b):
         _, alpha, alpha_c, _ = analysis_result["popt"]
-        _, alpha_err, alpha_c_err, _ = analysis_result["popt_err"]
+        _, _, alpha_c_err, _ = analysis_result["popt_err"]
 
         # Calculate epc_est (=r_c^est) - Eq. (4):
-        epc_est = scale * (1 - alpha_c / alpha)
+        epc_est = scale * (1 - alpha_c)
+        epc_est_err = scale * alpha_c_err
+        analysis_result["EPC"] = epc_est
+        analysis_result["EPC_err"] = epc_est_err
+
         # Calculate the systematic error bounds - Eq. (5):
-        systematic_err_1 = scale * (abs(alpha - alpha_c / alpha) + (1 - alpha))
+        systematic_err_1 = scale * (abs(alpha - alpha_c) + (1 - alpha))
         systematic_err_2 = (
             2 * (nrb * nrb - 1) * (1 - alpha) / (alpha * nrb * nrb)
             + 4 * (np.sqrt(1 - alpha)) * (np.sqrt(nrb * nrb - 1)) / alpha
         )
         systematic_err = min(systematic_err_1, systematic_err_2)
         systematic_err_l = epc_est - systematic_err
         systematic_err_r = epc_est + systematic_err
-
-        alpha_err_sq = (alpha_err / alpha) ** 2
-        alpha_c_err_sq = (alpha_c_err / alpha_c) ** 2
-        epc_est_err = (
-            ((nrb - 1) / nrb) * (alpha_c / alpha) * (np.sqrt(alpha_err_sq + alpha_c_err_sq))
-        )
-
-        analysis_result["EPC"] = epc_est
-        analysis_result["EPC_err"] = epc_est_err
         analysis_result["systematic_err"] = systematic_err
         analysis_result["systematic_err_L"] = systematic_err_l
         analysis_result["systematic_err_R"] = systematic_err_r
-        analysis_result["plabels"] = ["A", "alpha", "alpha_c", "B"]
-
-        if plot:
-            ax = plotting.plot_curve_fit(fit_fun_standard, analysis_result, ax=ax)
-            ax = plotting.plot_curve_fit(fit_fun_interleaved, analysis_result, ax=ax)
-            ax = plotting.plot_scatter(std_xdata, std_ydata, ax=ax)
-            ax = plotting.plot_scatter(int_xdata, int_ydata, ax=ax)
-            ax = plotting.plot_errorbar(std_xdata, std_ydata, std_ydata_sigma, ax=ax)
-            ax = plotting.plot_errorbar(int_xdata, int_ydata, int_ydata_sigma, ax=ax)
+
+        if plot and plotting.HAS_MATPLOTLIB:
+            ax = plotting.plot_curve_fit(fit_fun_standard, analysis_result, ax=ax, color="blue")
+            ax = plotting.plot_curve_fit(
+                fit_fun_interleaved,
+                analysis_result,
+                ax=ax,
+                color="green",
+            )
+            ax = self._generate_multi_scatter_plot(series_raw, x_raw, y_raw, ax=ax)
+            ax = self._generate_multi_errorbar_plot(series, xdata, ydata, ydata_sigma, ax=ax)
             self._format_plot(ax, analysis_result)
-            analysis_result.plt = plotting.pyplot
+            ax.legend(loc="center right")
+            figures = [ax.get_figure()]
+        else:
+            figures = None
+        return analysis_result, figures
+
+    @staticmethod
+    def _generate_multi_scatter_plot(series, xdata, ydata, ax):
```

Comment:
Note that the points of the standard and interleaved data are denoted the same (green x), only the fitting curve and mean and bars have separate colors